en/lessons/parsers/introduction.md

```diff
@@ -0,0 +1,62 @@
+---
+version: 1.0.0
+title: Introduction to Parser Combinators
+---
+
+{% include toc.html %}
+
+Parsers are ubiquitous in Haskell litterature because, thanks to Parser Combinators it illustrates the power of composition.
+
+Parsers, while being mostly associated to compilers and interpreters, can be found everywhere.
+
+Their goal is to take a input data (usually a text or a stream), and produce a structured output (usually a data structure, such as an [AST](https://en.wikipedia.org/wiki/Abstract_syntax_tree)), see [this article](https://lexi-lambda.github.io/blog/2019/11/05/parse-don-t-validate/) for day to day application.
```

Comment:
```suggestion
Their goal is to take input data (usually text or a stream), and produce structured output (usually a data structure, such as an [AST](https://en.wikipedia.org/wiki/Abstract_syntax_tree)). See [this article](https://lexi-lambda.github.io/blog/2019/11/05/parse-don-t-validate/) for day to day application.
```
I have removed "a" a couple times here: I believe your language, especially when referring to types, was grammatical, but in my opinion this sounds more natural. 

I also replaced a comma with a period to avoid the grammatical issue of comma splicing. 

This is likely a later consideration in the review process, but it seems a good idea to get permission from the author to link the article and, if this occurs, to ask her how she wants the article clarified.